templates/webapi/test/link_context.html.ep

```diff
@@ -0,0 +1,7 @@
+% layout 'codemirror';
+
+% content_for 'src_code' => begin
+  File Path:
```

Comment:
Using a layout was not like I imagined the unification but I suppose it is good as well.

However, now another nitpicky inconsistency becomes apparent: "Test-Module" vs. "File Path"  
I personally would prefer "Test module" and "File path".

Reply:
no problem from my side. i modified them accordingly with the suggestion.